### PR TITLE
Fixed SDK identifier for tvOS

### PR DESCRIPTION
**Pull request by regexident**

### CODE REVIEW

File: UniversalFramework_Base.xcconfig [hunk 1]
```diff
@@ -12,7 +12,7 @@ VALID_ARCHS[sdk=iphoneos*]             = arm64 armv7 armv7s
 VALID_ARCHS[sdk=iphonesimulator*]      = i386 x86_64
 VALID_ARCHS[sdk=watchos*]              = armv7k
 VALID_ARCHS[sdk=watchsimulator*]       = i386
-VALID_ARCHS[sdk=appletv*]              = arm64
+VALID_ARCHS[sdk=appletvos*]              = arm64
```
Comment (mrackwitz) [inline UniversalFramework_Base.xcconfig]: Ooops, thanks. But now the values are not on the same column anymore.

Comment (regexident) [inline UniversalFramework_Base.xcconfig]: Woops, fixed.

**Comment by mrackwitz** [comment]
👍

**Comment by regexident** [comment]
You were in [good company](https://github.com/apple/swift-package-manager/pull/411) with in these typos, btw. :grin:
At least xcconfigs didn't infect >47 projects with it. :frowning:

**Pull request merged**